### PR TITLE
[docs](partition) Improve auto partitions document

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

File: docs/en/docs/faq/data-faq.md [hunk 1]
```diff
@@ -169,9 +169,14 @@ FROM kafka
...
 ### Q12. ERROR 1105 (HY000): errCode = 2, detailMessage = (192.168.90.91)[CANCELLED][INTERNAL_ERROR]error setting certificate verify locations:  CAfile: /etc/ssl/certs/ca-certificates.crt CApath: none
 
 ```
 yum install -y ca-certificates
 ln -s /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt /etc/ssl/certs/ca-certificates.crt
 ```
+
+### Q13. create partition failed. partition numbers will exceed limit variable max_auto_partition_num
+
+To prevent accidental creation of too many partitions when importing data for auto-partitioned tables, we use the FE configuration item `max_auto_partition_num` to control the maximum number of partitions to be created automatically for such tables. If you really need to create more partitions, please modify the FE master node config file to increase the value and restart the node.
```
Comment (HappenLee) [inline docs/en/docs/faq/data-faq.md:182]: ready need restart ？

**Comment by zclllyybb** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by LemonLiTree — APPROVED** [review]
LGTM

**Pull request merged**